packages/components/src/Chips/InternalChipDismissible/hooks/useInternalChipDismissibleInput.tsx

```diff
@@ -67,6 +69,7 @@ export function useInternalChipDismissibleInput({
     handleReset: () => {
       setActiveIndex(activeIndex === 0 ? activeIndex : activeIndex - 1);
       setSearchValue("");
+      actions.handleCloseMenu();
```

Comment:
I don't think this is safe to move into here.

we call `handleReset` in `handleSelectOption`, so this will cause the menu to close when we make a selection.

Reply:
though in the scenario where we only want to have the menu open when there's a search term, `setSearchValue("")` will also close it haha - well that's if we use `!!searchValue` as a criteria anyway. we can make it more explicit if we'd like.